migration_steps/transform_casrec/transform/app/entities/reporting/annual_report_type_assignments.py

```diff
@@ -2,59 +2,176 @@
 from utilities.basic_data_table import get_basic_data_table
 import logging
 import os
+import pandas as pd
 
-from helpers import get_mapping_dict
+from helpers import get_mapping_dict, get_table_def
+from transform_data.apply_datatypes import reapply_datatypes_to_fk_cols
+from transform_data.unique_id import add_unique_id
 
 log = logging.getLogger("root")
-#
-# definition = {
-#     "source_table_name": "account",
-#     "source_table_additional_columns": [],
-#     "destination_table_name": "annual_report_log",
-# }
-#
-# mapping_file_name = "annual_report_type_assignments"
+
+
+# Until we work out what to do with the type column
+DEFAULT_REPORT_TYPE = "-"
+
+
+# :param row: single row with ord_stat and ord_risk_lvl columns;
+#      this will be the row from the orders for this annual report log
+#      with the latest date
+# :returns: reporttype value
+def _calculate_reporttype(row):
```

Comment:
Think we may need to tighten up so it returns none on things that don't match 1,2,2a,3 here :)

Reply:
Done